src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/EventBuilderImpl.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Reflection.Metadata;
+
+namespace System.Reflection.Emit
+{
+    internal sealed class EventBuilderImpl : EventBuilder
+    {
+        private string _name;
```

Comment:
Can this and the next 3 fields be readonly?

Reply:
yes, the can